de-de/clojure-de.html.markdown

```diff
@@ -0,0 +1,423 @@
+
+; Verwende require um ein Modul zu importieren
+(require 'clojure.string)
+
+; Verwende / um eine Funktion aus einem Modul aufzurufen
+; Hier verwenden wir das Modul clojure.string und die Funktion blank?
+(clojure.string/blank? "") ; => true
+
+; Du kannst auch einem Modul einem kürzerern Namen beim Import geben
+(require '[clojure.string :as str])
+(str/replace "Das ist ein Test." #"[a-o]" str/upper-case) ; => "DAs IsT EIN TEsT."
+; (#"" bezeichnet einen regulären literalen Ausdruck)
+
+; Du kannst require aus einem Namespace verwenden (auch use ist möglich, aber nicht zu empfehlen)
+; indem du :require verwendest.
+; Du brauchst keine Zitierzeichen für deine Module verwenden, wenn du
+; es auf diese Weise machst.
+(ns test
+  (:require
+    [clojure.string :as str]
+    [clojure.set :as set]))
+
+; Java
+;;;;;;;;;;;;;;;;;
+
+; Java hat eine riesige und nützliche Standardbibliothek,
+; du möchtest lernen wie man sie verwendet.
+
+; Verwende import um ein Java modul zu laden.
+(import java.util.Date)
+
+; Du kannst auch von einem ns importieren.
+(ns test
+  (:import java.util.Date
+           java.util.Calendar))
+
+; Verwende den Klassennamen mit einem "." am Ende, um eine neue Instanz zu erstellen
+(Date.) ; <a date object>
+
+; Verwende . um Methoden aufzurufen oder verwende die ".method" Abkürzung
+(. (Date.) getTime) ; <a timestamp>
+(.getTime (Date.)) ; Genau das Selbe
+
+; Verwende / um statische Methoden aufzurufen
+(System/currentTimeMillis) ; <a timestamp> (system ist immer da)
+
+; Verwende doto um mit veränderliche Klassen besser umzugehen
+(import java.util.Calendar)
+(doto (Calendar/getInstance)
+  (.set 2000 1 1 0 0 0)
+  .getTime) ; => A Date. set to 2000-01-01 00:00:00
+
+; STM
+;;;;;;;;;;;;;;;;;
+
+; Software Transactional Memory ist der Mechanismus, den Clojure verwendet
+; um mit persistenten Zuständen umzugehen. Es gibt ein Paar Konstrukte in
+; Clojure die es verwenden.
+
+; Ein Atom ist das Einfachste. Gebe es einen Initalwert
+(def my-atom (atom {}))
+
+; Update ein Atom mit swap!.
+; swap! nimmt eine Funktion und ruft sie mit dem aktuellen Zustand des
+; Atoms auf und alle nachfolgenden Argumente als das Zweite
+(swap! my-atom assoc :a 1) ; Setzt my-atom zu dem Ergebnis von (assoc {} :a 1)
+(swap! my-atom assoc :b 2) ; Setzt my-atom zu dem Ergebnis von (assoc {:a 1} :b 2)
+
+; Verwende '@' um das Atom zu dereferenzieren und den Wert zu bekommen
+my-atom  ;=> Atom<#...> (Gibt das Atom Objekt zurück
+@my-atom ; => {:a 1 :b 2}
+
+; Hier ist ein einfacher Zähler mit einem Atom
+(def counter (atom 0))
+(defn inc-counter []
+  (swap! counter inc))
+
+(inc-counter)
+(inc-counter)
+(inc-counter)
+(inc-counter)
+(inc-counter)
+
+@counter ; => 5
+
+; Andere STM Konstrukte sind refs und agents.
+; Refs: http://clojure.org/refs
+; Agents: http://clojure.org/agents
+```
+
+### Weiterführende Literatur
+
+Das ist alles andere als erschöpfend, aber hoffentlich ist es genug, um dich auf die Beine zu stellen.
```

Comment:
Unfortunately, this is too literal for a translation, both "erschöpfend" and "dich auf die Beine stellen" don't quite work here

Reply:
If you come up with an alternative, please submit a PR.